doc/grafana.md

```diff
@@ -1,5 +1,8 @@
 # Grafana
 
+>[!warning] ❌ Broken on Darwin
+> See here: https://github.com/juspay/services-flake/pull/204#issuecomment-2145497680
+
```

Comment:
Since this appears to be a transient issue I wouldn't give a scary warning to the user (it is no fault of services-flake anyway; just nixpkgs'). But we can mention the nixpkgs revision grafana works for macOS in a footnote or something.